naples/routes/metadata.py

```diff
@@ -7,83 +7,110 @@
 from naples.dependency import get_current_user, get_admin
 from naples.logger import log
 
-metadatas_router = APIRouter(prefix="/metadatas", tags=["Metadatas"])
+metadatas_router = APIRouter(prefix="/metadata", tags=["Metadata"])
 
 
 @metadatas_router.get(
-    "/",
+    "/keys",
```

Comment:
This is wrong. You should not return keys. You should return a pydantic schema, which already has all the keys frontend is asking